interfaces/builtin/acrn.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021 Intel Corporation, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package builtin
+
+const acrnSummary = `allows access to the ACRN device`
+
+const acrnBaseDeclarationSlots = `
+  acrn:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const acrnConnectedPlugAppArmor = `
+# Description: Allow access to resources required by ACRN.
+#   allow offline CPU cores
+/sys/devices/system/cpu/cpu[0-9]*/online w,
+
+#   allow write access to ACRN Virtio and Hypervisor service Module
+/dev/acrn_vhm rw,
```

Comment:
This will probably need an udev rule.

Reply:
@bboozzoo thanks for the review comments.
As the VHM module is static compiled in ACRN kernel, do we still require udev rules?

Reply:
Try running `udevadm info /dev/acrn_vhm`. Is there any information about the device? The udev rules are used to tag devices which can be accessed by given snap. When the snap application runs we set up a device cgroup and allow access to select devices only.

Reply:
@bboozzoo thanks for your quick response. Below is the output from `udevadm info /dev/acrn_vhm`:
```
P: /devices/virtual/vhm/acrn_vhm
N: acrn_vhm
E: DEVNAME=/dev/acrn_vhm
E: DEVPATH=/devices/virtual/vhm/acrn_vhm
E: MAJOR=243
E: MINOR=0
E: SUBSYSTEM=vhm
E: USEC_INITIALIZED=3905750
E: net.ifnames=0
```
Would you advise what should we put in `acrnConnectedPlugUDev`?

Reply:
@bboozzoo , I've referenced the `time-control` interface and add an udev rule for it, would you please advice whether it's applicable? Thanks.